JS-Algorithms/challenges.js

```diff
@@ -70,7 +87,15 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  // YOUR CODE HERE...
+  let sum = 0n;
+
+  for (let i = 1; i <= number; i++){
+    let pow = BigInt(i) ** BigInt(i);
```

Comment:
can we use this instead of declaring the pow variable?

```suggestion
    sum += BigInt(i) ** BigInt(i);
```